src/server/data/collection.js

```diff
@@ -698,6 +700,12 @@ export class QDataCollection {
         if (this.isTests) {
             return;
         }
+        if (this.provider.isHotUpdate) {
```

Comment:
We must not start providers hot update from this point. We must start it from QData.dropCachedDbInfo (data.js):
this.providers.immutable.hotUpdate()

Reply:
Where should I call `hotUpdate()` for the first time at the start to init fingerprint?

Reply:
QData.start